packages/core/src/App/Containers/Layout/header/default-header.jsx

```diff
@@ -226,4 +229,5 @@ export default connect(({ client, common, ui, notifications }) => ({
     is_trading_assessment_for_existing_user_enabled: ui.is_trading_assessment_for_existing_user_enabled,
     is_landing_company_loaded: client.is_landing_company_loaded,
     is_switching: client.is_switching,
+    setTogglePlatformType: traders_hub.setTogglePlatformType,
```

Comment:
please add this action into the packages/stores/types.ts and `packages/stores/src/mockStore.ts`